packages/utils/odsp-doclib-utils/package.json

```diff
@@ -25,7 +55,7 @@
 	},
 	"main": "dist/index.js",
 	"module": "lib/index.mjs",
```

Comment:
I think it's safe to remove the module field now, and Jason prefers that I think. But there are enough still present that we'll have to do a cleanup pass at some point regardless.

Reply:
FYI - I tried removing it, but 'policy-check' complained, so I put it back.